src/pages/VoterDetail/History/index.tsx

```diff
@@ -0,0 +1,85 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import { Typography } from '@mui/material';
+import { Pagination, Spinner } from 'components';
+import { useTranslation } from 'translation';
+import { IVoterHistory } from 'types';
+import VoterProposal from './VoterProposal';
+import { useStyles } from './styles';
+
+interface IHistoryProps {
+  className?: string;
+  voterHistory: IVoterHistory[] | undefined;
+  setCurrentPage: (page: number) => void;
+  total: number;
+  limit: number;
+  isFetching: boolean;
+}
+
+export const History: React.FC<IHistoryProps> = ({
+  className,
+  voterHistory,
+  setCurrentPage,
+  total,
+  limit,
+  isFetching,
+}) => {
+  const styles = useStyles();
+  const { t } = useTranslation();
+  return (
+    <div className={className}>
+      <Typography variant="h4">{t('voterDetail.votingHistory')}</Typography>
+      {voterHistory && !isFetching ? (
+        voterHistory.map(
+          ({
+            proposal: {
+              id,
+              description,
+              state,
+              endDate,
+              forVotesWei,
+              abstainedVotesWei,
+              againstVotesWei,
+              createdDate,
+              queuedDate,
+              cancelDate,
+              executedDate,
+            },
+            support,
+          }) => (
+            <VoterProposal
+              key={id}
+              proposalNumber={id}
+              proposalTitle={description.title}
+              proposalState={state}
+              forVotesWei={forVotesWei}
+              againstVotesWei={againstVotesWei}
+              abstainedVotesWei={abstainedVotesWei}
+              userVoteStatus={support}
+              createdDate={createdDate}
+              queuedDate={queuedDate}
+              cancelDate={cancelDate}
+              executedDate={executedDate}
+              endDate={endDate}
+            />
+          ),
+        )
+      ) : (
+        <Spinner />
+      )}
```

Comment:
I think we should always display the list of `voterHistory` isn't empty and display the loader when `isFetching` is truthy, because currently the page blinks every time it loads new content.

This would also make the UX consistent with the other pages (e.g.: History).